deploy/addons/olm/olm.yaml.tmpl

```diff
@@ -78,9 +78,9 @@ spec:
           command:
           - /bin/olm
           args:
-          - -namespace
+          - --namespace
```

Comment:
have u verified that the extra - is needed ? was not working before without it ?

Reply:
I think for OLM 0.17.0 should be doube `--`
olm --version
OLM version: 0.17.0
git commit: 5eb1fa6ad6ecfbf1263a145e54e5bd181a078097

olm --help
Usage of olm:
      --debug                                     use debug log level
      --feature-gates mapStringBool               A set of key=value pairs that describe feature gates for alpha/experimental features. Options are:
                                                  AllAlpha=true|false (ALPHA - default=false)
                                                  AllBeta=true|false (BETA - default=false)
      --interval duration                         wake up interval (default 5m0s)
      --namespace string                          namespace where cleanup runs
      --profiling                                 serve profiling data (on port 8080)
      --tls-cert string                           Path to use for certificate key (requires tls-key)
      --tls-key string                            Path to use for private key (requires tls-cert)
      --version                                   displays olm version
      --watchedNamespaces -watchedNamespaces=""   comma separated list of namespaces for olm operator to watch. If not set, or set to the empty string (e.g. -watchedNamespaces=""), olm operator will watch all namespaces in the cluster.
      --writePackageServerStatusName string       ClusterOperator name in which to write status for package API server, set to "" to disable.
      --writeStatusName string                    ClusterOperator name in which to write status, set to "" to disable.